4-4-4.py

```diff
@@ -0,0 +1,44 @@
+'''
+Больше среднего
+
+Напишите программу, которая выводит количество элементов квадратной матрицы в каждой строке, больших среднего арифметического элементов данной строки.
+
+Формат входных данных
+На вход программе подаётся натуральное число nnn — количество строк и столбцов в матрице, затем элементы матрицы (целые числа) построчно через пробел.
+
+Формат выходных данных
+Программа должна вывести nnn чисел — для каждой строки количество элементов матрицы, больших среднего арифметического элементов данной строки.
+Тестовые данные 🟢
+
+Sample Input 1:
+
+4
+1 2 3 4
+5 6 3 15
+0 2 3 1
+5 2 8 5
+
+Sample Output 1:
+
+2
+1
+2
+1
+'''
+
+def above_average(m_size):
+    matrix = []
+    result = 0
```

Comment:
Keep variables closer as much as possible to their first usage. This improves readability.

Reply:
let's talk on the phone